templates/cluster-template.yaml

```diff
@@ -123,6 +123,17 @@ spec:
       templateID: ${TEMPLATE_VMID}
       format: "qcow2"
       full: true
+      numSockets: ${NUM_SOCKETS:=2}
```

Comment:
I removed the xl-nodes template cause our usage docs mention the customization of the machine while not supported by the default template.